### PR TITLE
Paul/update docs

### PR DESCRIPTION
**Pull request by yoid2000**

### CODE REVIEW

File: docs/admin_guide.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Extension behavior is controlled by __security labels__ and __settings__. Securi
 
 Only superusers can assign security labels. To remove a security label from an object, set it to `NULL`.
 
-The command `SELECT * FROM diffix.show_labels();` displays the current security labels assigned to tables and columns by the extension.
+The command `SELECT * FROM diffix.show_labels();` displays the current security labels assigned to tables and columns by the extension. Note that if this command fails, then probably `pg_diffix` has not been enabled for the database. See the [Administration Tutorial](admin_tutorial.md) for step-by-step instructions.
```
Comment (cristianberneanu) [inline docs/admin_guide.md:15]: If `diffix.show_labels()` fails (the function is not found), it means `pg_diffix` has not been enabled for the current database. If it returns an empty result set, it means no data or users have been configured.
If `diffix.show_settings()` returns nothing, then the extension is not loaded for the current session.

File: docs/admin_tutorial.md [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
-named `test_db` was created, and the personal data is in the table `test_table`,
-containing a column named `id`, which uniquely identifies protected entities (the anonymization ID).
+This document provides examples on how to install and configure `pg_diffix` to expose a simple dataset for anonymized querying.  It assumes that an existing installation of PostgreSQL 14 on a Linux system is available.
 
-## Installation
+It provides multiple examples. The first is a [simple example](#simple-example) that assumes a database named `test_db` was created, and the personal data is in the table `test_table`, containing a column named `id`, which uniquely identifies protected entities (the anonymization ID).
+
+Following this are [several examples of real datasets](#real-examples) which discuss the rationale behind how to determine which columns contain anonymization IDs.
+
+# Simple Example
```
Comment (cristianberneanu) [inline docs/admin_tutorial.md:9]: We already have a top heading. This should have `##` and the following headings should have `###`.

File: docs/admin_tutorial.md [hunk 3]
```diff
@@ -46,32 +48,389 @@ CREATE EXTENSION pg_diffix;
 ALTER DATABASE test_db SET session_preload_libraries TO 'pg_diffix';
 ```
 
-## Configuration
+## User configuration (per database)
```
Comment (cristianberneanu) [inline docs/admin_tutorial.md:51]: User configuration is done globally, not per database.

Comment (cristianberneanu) [reply docs/admin_tutorial.md:51]: I mean the creation of the account and the labelling, the access rights have to be done per database (so better move them to the next section).

File: docs/admin_tutorial.md [hunk 4]
```diff
@@ -46,32 +48,389 @@ CREATE EXTENSION pg_diffix;
...
+ gender           | text   
+ ssn              | text   
+ email            | text   
+ street           | text   
+ zip              | text   
+ ```
+
+ The first 6 columns are from the original `accounts` table. Subsequent columns come from the JOIN with `clients`. (The last 8 columns are synthetically generated to mimic personally identifiable information for the bounty program.)
+
+ Note that the `uid` column was originally called `client_id`. It was modified to `uid` for the purpose of the Diffix bounty programs run in prior years.
```
Comment (cristianberneanu) [inline docs/admin_tutorial.md:115]: This section doesn't seem relevant for the purpose of this document.

File: docs/admin_tutorial.md [hunk 5]
```diff
@@ -46,32 +48,389 @@ CREATE EXTENSION pg_diffix;
...
 ```
 CALL diffix.mark_personal('test_table', 'id');
 ```
 
-2\. Create an account for the analyst:
+__That's it!__ The analyst can now connect to the database and issue (only) anonymizing queries against the test dataset. Refer to the [Administration Guide](admin_guide.md) for additional details, or if you wish to modify default anonymization parameters.
+
+# Real Examples
+
+The real examples are databases that are used for the Diffix bounty programs. The databases can be found at https://db001.gda-score.org/.
```
Comment (edongashi) [inline docs/admin_tutorial.md:84]: To keep the tutorial short and not overwhelming, what do you think of moving this section to a separate file?

**Comment by cristianberneanu** [comment]
These changes should target the `release/elm` branch.
Though that might require a messy rebase now, so, alternatively, you can recreate them in a duplicate PR.

**Comment by yoid2000** [comment]
How do I target the `release/elm` branch? I'm happy to duplicate it, but not sure how to make the appropriate branch

**Comment by edongashi** [comment]
> How do I target the `release/elm` branch? I'm happy to duplicate it, but not sure how to make the appropriate branch

I created a new branch `paul/update-docs-elm` which is based on `release/elm`. You can edit this PR to change _both_ base and target branch or create a new one.

**Comment by cristianberneanu** [comment]
> How do I target the `release/elm` branch? I'm happy to duplicate it, but not sure how to make the appropriate branch

Edon already did it for you, but in case you need to do it again in the future:

One way would be to create a new branch from `release/elm`, add your changes to it (either recreate them manually or using `cherry-pick`) and create a new PR which targets `release/elm` instead of `master`:
```sh
git checkout release/elm
git pull
git checkout -B paul/update-docs-new
git cherry-pick 78d81d3..fd78ac8 # cherry pick all commits from this PR, might create merge conflicts
git push 
```

Alternatively, you could rebase the current branch over `release/elm` by deleting all commits you didn't add:
```sh
git checkout release/elm
git pull
git checkout paul/update-docs-new
git rebase -i release/elm # edit the rebase list by deleting commits from `master` and solve any merge conflicts
git push -f # this overwrites the commits in this PR
```
then edit this PR and change the target branch to `release/elm`.

**Comment by yoid2000** [comment]
> I created a new branch `paul/update-docs-elm` which is based on `release/elm`. Looks like it needs a new pull request.

So as I understand it, I can just checkout `paul/update-docs-elm`, and start applying your comments to that branch?

**Comment by edongashi** [comment]
> I don't understand what it means to "target `release/elm`". What I did was simply checkout branch `paul/update-docs/elm`. Is there something else I need to do?

Sorry, I mixed two things in the same sentence. No, simply checkout `paul/update-docs-elm`.

When opening the pull request on GitHub choose `release/elm` instead of `master` from the dropdown.

**Pull request closed without merging**